include/deal.II/fe/mapping_q_cache.h

```diff
@@ -111,6 +112,38 @@ class MappingQCache : public MappingQGeneric<dim, spacedim>
                const typename Triangulation<dim, spacedim>::cell_iterator &)>
                &compute_points_on_cell);
 
+  /**
+   * Initialize the data cache by computing the mapping support points for all
+   * cells (on all levels) of the given triangulation and a given @p mapping
+   * and transforming these points via the function @p transformation_function.
+   *
+   * If @p is_displacement_function is set to true, the return value of the
+   * function is interpreted as relative deformation and the result is added
+   * to the original point.
```

Comment:
Related to the other comment and independently of which logic is implemented as default, I believe both states (true/false) should be explained explicitly. Currently, the default value is false but the docu explains the true-case.